actions/index.ts

```diff
@@ -0,0 +1,48 @@
+export enum ActionType {
```

Comment:
This file types out the various actions the game can do. There are empty functions at the bottom that we'll eventually use to update state. Since we're using a [Context](https://reactjs.org/docs/context.html) we might not need those functions to update state.